apps/ml/app/main.py

```diff
@@ -0,0 +1,22 @@
+from fastapi import FastAPI, Request
+from fastapi.responses import JSONResponse
+from pydantic import BaseModel
+
+app = FastAPI()
+
+
+class ActivityTrackingMeta(BaseModel):
+    type: str
+    interval: str
+    repeats: int
+
+@app.post("/api/v1/load_data")
+async def classify_data(info : Request):
+    req_info = await info.json()
+
+# TODO pass req_info to clasificator and return content for response
+    json_response = {
+      "ActivityTrackingMeta": ActivityTrackingMeta
+    }
+
+    return JSONResponse(content=json_response)
```

Comment:
Instead of using plain JSONResponse you should be able to use an output model, so check here: https://fastapi.tiangolo.com/tutorial/response-model/ how do they type the response model. And guess what, we already have the `ActivityTrackingMeta` class 😎 , we just need to adjust the decorator to support proper input and output models 😄